src/iccpd/src/iccp_cmd_show.c

```diff
@@ -63,7 +65,11 @@ int iccp_mclag_config_dump(char * *buf,  int *num, int mclag_id)
 
     LIST_FOREACH(csm, &(sys->csm_list), next)
     {
-        memset(&state_info, 0, sizeof(struct mclagd_state));
+#ifdef __STDC_LIB_EXT1__
+    memset_s(&state_info, sizeof(state_info), 0, sizeof(struct mclagd_state));
```

Comment:
>     [](http://example.com/codeflow?start=0&length=4)

Please fix the indentation.

Reply:
I mean it should at 2-level indentation, ignore all the preprocessor lines.

Reply:
Got it, thanks!